app/Database/Seeds/UserSeeder.php

```diff
@@ -9,29 +10,28 @@ class UserSeeder extends Seeder
     public function run()
     {
         $faker = Factory::create();
-        $data = [
+        $data  = [
             [
-                "userFirstName" => $faker->name(),
-                "userLastName" => $faker->lastName(),
-                "userEmail" => "admin@admin.com",
-                "userPassword" => password_hash("fake_password", PASSWORD_BCRYPT),
-                "userToken" => "FAS12AQajqa",
-                "confirmed" => 0,
-                "isAdmin" => 1
+                'userFirstName' => $faker->name(),
+                'userLastName'  => $faker->lastName(),
+                'userEmail'     => 'admin@admin.com',
+                'userPassword'  => password_hash('fake_password', PASSWORD_BCRYPT),
+                'userToken'     => 'FAS12AQajqa',
+                'isConfirmed'   => 1,
```

Comment:
Based on what you discussed with Marcelo this field now should be called only `confirmed`. Cause this seed is not working. 
```suggestion
                'confirmed'   => 1,
```